lib/v2/qdu/jwc.js

```diff
@@ -0,0 +1,60 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const wait = require('@/utils/wait');
+
+const base = 'https://jwc.qdu.edu.cn/';
+
+module.exports = async (ctx) => {
+    const response = await got({
+        method: 'get',
+        url: `${base}jwtz.htm`,
+    });
+
+    const $ = cheerio.load(response.data);
+    const list = $('.notice_item').children();
+    const items = await Promise.all(
+        list.map((i, item) => {
+            item = $(item);
+            const itemTitle = item.find('.active').text();
+            const itemDate = item.find('span').text();
+            const path = item.find('.active').attr('href');
+            let itemUrl = '';
+            if (!path.startsWith('http')) {
+                itemUrl = base + path;
+            } else {
+                itemUrl = path;
+            }
+            return ctx.cache.tryGet(itemUrl, async () => {
+                let description = '';
+                if (!path.startsWith('https://') && !path.startsWith('http://')) {
+                    await wait(1500);
```

Comment:
You can remove this `wait` since the 404 https://github.com/DIYgod/RSSHub/pull/12273#issuecomment-1501370211 is not caused by rate limiting but a firewall rule as those items can only be visited through **intranet** https://github.com/DIYgod/RSSHub/actions/runs/4654402500/jobs/8236105858#step:11:20

Reply:
> You can remove this `wait` since the 404 [#12273 (comment)](https://github.com/DIYgod/RSSHub/pull/12273#issuecomment-1501370211) is not caused by rate limiting but a firewall rule as those items can only be visited through **intranet** [DIYgod/RSSHub/actions/runs/4654402500/jobs/8236105858#step:11:20](https://github.com/DIYgod/RSSHub/actions/runs/4654402500/jobs/8236105858#step:11:20)

Confirmed.